guides/common/modules/con_alternate-content-sources-overview.adoc

```diff
@@ -0,0 +1,7 @@
+[id="Alternate_Content_Sources_Overview_{context}"]
+= Alternate Content Sources Overview
+
+Alternate content sources define alternate paths to download content during synchronization.
+This speeds up the repository synchronization.
```

Comment:
This is not always the case. Maybe reword to something like "depending on your infrastructure" or similar. Or maybe physical distance or network connection?